### PR TITLE
Remove trailing open parenthesis in tools.proj

### PR DESCRIPTION
**Pull request by riarenas**
This was lost in https://github.com/dotnet/arcade/commit/aceee7d96c6554b410f50ec1a8ed2f1464c171c6#diff-929b1158791b608c6a417f92fa7b92ea993c30c4089b7d8a67a71368fda16594R47

This was caught by arcade-validation: https://github.com/dotnet/arcade-validation/pull/4572

### To double check:

* [ ] The right tests are in and the right validation has happened.  Guidance: https://github.com/dotnet/arcade/blob/main/Documentation/Validation.md

### CODE REVIEW

File: src/Microsoft.DotNet.Arcade.Sdk/tools/Tools.proj [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
   <PropertyGroup>
     <_ImportOrUseTooling>false</_ImportOrUseTooling>
-    <_ImportOrUseTooling Condition="('$(DotNetBuildSourceOnly)' != 'true' or ('$(DotNetBuildPhase)' == 'Repo' and '$(DotNetBuildOrchestrator)' != 'true')">true</_ImportOrUseTooling>
+    <_ImportOrUseTooling Condition="('$(DotNetBuildSourceOnly)' != 'true' or ('$(DotNetBuildPhase)' == 'Repo' and '$(DotNetBuildOrchestrator)' != 'true'))">true</_ImportOrUseTooling>
```
Comment (mmitche) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/Tools.proj:47]:
```suggestion
    <_ImportOrUseTooling Condition="'$(DotNetBuildSourceOnly)' != 'true' or ('$(DotNetBuildPhase)' == 'Repo' and '$(DotNetBuildOrchestrator)' != 'true')">true</_ImportOrUseTooling>
```

**Review by mmitche — APPROVED** [review]

**Comment by dougbu** [comment]
nit: This PR and the final commit removed a _leading_ (and unnecessary) open parenthesis. Title says the opposite, which is a bit confusing&hellip;

**Comment by riarenas** [comment]
Hmm is the confusion that I should have changed the commit to say removed leading open parenthesis instead of trailing? Apologies, must be an English nuance that I messed up.

**Comment by dougbu** [comment]
> is the confusion that I should have changed the commit to say "removed leading open parenthesis" instead of using the word trailing?

exactly

**Comment by riarenas** [comment]
Thanks. I will strive to perfect my English going forward so that there is no confusion

**Pull request merged**